src/app_model/types/_keys/_keybindings.py

```diff
@@ -117,6 +169,33 @@ def to_int(self, os: Optional[OperatingSystem] = None) -> int:
             mods |= KeyMod.CtrlCmd if os.is_mac else KeyMod.WinCtrl
         return mods | (self.key or 0)
 
+    def _kb2mods(self) -> List[str]:
+        """Extract list of modifiers from this SimpleKeyBinding."""
+        mods = []
+        if self.ctrl:
+            mods.append("Ctrl")
+        if self.shift:
+            mods.append("Shift")
+        if self.alt:
+            mods.append("Alt")
+        if self.meta:
+            mods.append("Meta")
+        return mods
+
+    def to_text(
+        self, os: Optional[OperatingSystem] = None, use_symbols: bool = False
+    ) -> str:
+        """Get a user-facing string representation of this SimpleKeyBinding.
+
+        Optionally, the string representation can be constructed with symbols
+        like ↵ for Enter or OS specific ones like ⌘ for Cmd on MacOS.
+        """
+        joinchar, key_symbols, key_names = get_keys_user_facing_representation(os=os)
```

Comment:
i think @jni wants more control over the joinchar (see #210) so joinchar should be a parameter with a default fallback if not provided